test/proxy/transparent/ProxyAdmin.test.js

```diff
@@ -51,18 +39,10 @@ contract('ProxyAdmin', function (accounts) {
 
     it('changes proxy admin', async function () {
       await this.proxyAdmin.changeProxyAdmin(this.proxy.address, newAdmin, { from: proxyAdminOwner });
-      expect(await this.proxy.admin.call({ from: newAdmin })).to.eq(newAdmin);
-    });
-  });
+      const proxyAdminSlot = await getSlot(this.proxy, AdminSlot);
+      const proxyAdminAddress = web3.utils.toChecksumAddress(proxyAdminSlot.substr(-40));
```

Comment:
Let's define a helper in `erc1967.js` called `getAddressInSlot` that does what these two lines do, so that we don't have to repeat them every time we need to read an address.